integration_tests/samples/socket_mode/bolt_oauth_aiohttp_example.py

```diff
@@ -0,0 +1,68 @@
+# ------------------
+# Only for running this script here
+import sys
+from os.path import dirname
+
+sys.path.insert(1, f"{dirname(__file__)}/../../..")
+# ------------------
+
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+import os
+
+from slack_bolt.app import App
+from slack_bolt.context import BoltContext
+from slack_bolt.oauth.oauth_settings import OAuthSettings
+
+app = App(
+    signing_secret=os.environ["SLACK_SIGNING_SECRET"],
+    oauth_settings=OAuthSettings(
+        client_id=os.environ["SLACK_CLIENT_ID"],
+        client_secret=os.environ["SLACK_CLIENT_SECRET"],
+        scopes=os.environ["SLACK_SCOPES"].split(","),
+    )
+)
+
+
+@app.event("app_mention")
+def mention(context: BoltContext):
+    context.say(":wave: Hi there!")
+
+
+@app.event("message")
+def message(context: BoltContext, event: dict):
+    context.client.reactions_add(
+        channel=event["channel"], timestamp=event["ts"], name="eyes",
+    )
+
+
+if __name__ == "__main__":
+
+    def run_socket_mode_app():
+        import asyncio
+        from bolt_adapter.aiohttp import SocketModeApp
+
+        async def socket_mode_app():
+            app_token = os.environ.get("SLACK_APP_TOKEN")
+            await SocketModeApp(app, app_token).connect_async()
+            await asyncio.sleep(float("inf"))
+
+        asyncio.run(socket_mode_app())
+
+    from concurrent.futures.thread import ThreadPoolExecutor
+
+    socket_mode_thread = ThreadPoolExecutor(1)
+    socket_mode_thread.submit(run_socket_mode_app)
```

Comment:
I know this is a bit tricky but if you want to mix asyncio based library with non-asyncio one, doing so is tricky by nature. If you don't server OAuth flow in the same app, everything is simple enough.